static/bidder-info/axonix.yaml

```diff
@@ -14,3 +14,7 @@ capabilities:
       - banner
       - video
       - native
+userSync:
+  redirect:
+    url: "https://openrtb-us-east-1.axonix.com/syn?supply={{.AccountID}}&redirect={{.RedirectURL}}"
+    userMacro: xxEMODO_IDxx
```

Comment:
This seems to be incorrect userSync. `AccountID` should not be provided in userSync URLs. Also prebid server build is failing with the following error -
```
**cannot create syncer for bidder axonix with key axonix: redirect template: axonix_usersync_url:1:50: executing "axonix_usersync_url" at <.AccountID>: can't evaluate field AccountID in type macros.UserSyncPrivacy**
```

Reply:
works now!